src/pages/wallet/Asset.tsx

```diff
@@ -74,7 +74,6 @@ const Asset = (props: Props) => {
                   decimals={decimals}
                   fixed={2}
                   denom=""
-                  token=""
                 />
```

Comment:
Decimals included.  Consider making `fixed` default to 2 inside of the `Read` component.

Reply:
fixed can both be a boolean and a number